robocop/run.py

```diff
@@ -8,18 +8,20 @@
 from robocop.config import Config
 from robocop import reports
 from robocop.utils import DisablersFinder, FileType, FileTypeChecker
+from robocop.exceptions import DuplicatedMessageError
 
 
 class Robocop:
-    def __init__(self):
+    def __init__(self, from_cli=False):
```

Comment:
thanks for this argument we can create Robocop instance inside unit test without cli (if we don't need it)